OrbitQt/processlauncherwidget.ui

```diff
@@ -27,12 +16,7 @@
     <number>0</number>
    </property>
    <item row="0" column="0">
-    <widget class="QSplitter" name="splitter">
-     <property name="orientation">
-      <enum>Qt::Vertical</enum>
-     </property>
-     <widget class="OrbitDataViewPanel" name="LiveProcessList" native="true"/>
-    </widget>
+    <widget class="OrbitDataViewPanel" name="LiveProcessList"/>
```

Comment:
I think you'll need that `native="true"` otherwise QtDesigner will add it when opened next time. I recently made all the UI-files compatible with QtDesigner. Have you tried opening them with designer? It would be great if we could preserve the compatibility.

Reply:
I made these changes with the designer, so I'm missing something.

Reply:
But I can just add `native="true"` back if you prefer.

Reply:
If the designer made these changes, I'm just wrong. I apologize. :innocent:

Reply:
> But I can just add `native="true"` back if you prefer.

Not at all. I was just the case for that designer always added that by itself.

Reply:
Maybe we just have a different setup? For example, I'm doing this on Linux.